src/js/gi/selectors/calculator.js

```diff
@@ -836,7 +836,7 @@ export const getCalculatedBenefits = createSelector(
         value: formatCurrency(derived.totalToYou)
       },
       perTerm: {
-        tuitionAndFees: {
+        tuitionFees: {
```

Comment:
Made this small change to make the "learn more" link creation smoother, as the section keys map directly to the fragment names for the links.